### PR TITLE
Updated readme

### PR DESCRIPTION
**Pull request by RN0311**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,7 +198,7 @@ found in `doc_deps.deb.txt`:
 
 	sudo apt-get install `cat doc_deps.deb.txt`
 
-### Building the Documentation
+### Documentation
```
Comment (joscollin) [inline README.md]: I'd either leave it unchanged or change it to something meaningful like `Build Instructions`.

Comment (RN0311) [reply README.md]: Sure! I'll do changes!
Thanks a bunch!

**Comment by joscollin** [comment]
Read:

https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#1-sign-your-work
https://github.com/ceph/ceph/blob/master/SubmittingPatches.rst#1-github-pull-request
https://github.com/todotxt/todo.txt-android/wiki/Squash-All-Commits-Related-to-a-Single-Issue-into-a-Single-Commit

**Comment by joscollin** [comment]
No response. So closing this.

**Pull request closed without merging**